python/examples/demo.py

```diff
@@ -27,25 +27,25 @@
 
 # test out various library functions after running in.demo
 
-spa.file("in.demo")
+spa.file(b"in.demo")
```

Comment:
@stefanfechter just noticed there are some extraneous ` b` characters, seems to only be in the demo.py file.